app-modules/alert/src/Observers/AlertObserver.php

```diff
@@ -53,9 +52,7 @@ public function created(Alert $alert): void
             // Creating the subscription directly so that the alert can be sent to this User as well
             resolve(SubscriptionCreate::class)->handle($user, $alert->getSubscribable(), false);
 
-            Cache::tags([match ($alert->concern_type) {
-                app(Contact::class)->getMorphClass() => "user-{$user->getKey()}-contact-alerts",
-            }])->flush();
+            Cache::tags(["user-{$user->getKey()}-contact-alerts"])->flush();
```

Comment:
This could have to do with this branch being not up-to-date with the base branch `main`. So perhaps rebasing this on `main` and force pushing up will resolve this.

If after you do that, this still doesn't have `{}` around it, please add that.

```suggestion
            Cache::tags(["{user-{$user->getKey()}-contact-alerts}"])->flush();
```